broker/src/test/java/org/apache/rocketmq/broker/topic/TopicValidatorTest.java

```diff
@@ -70,6 +64,62 @@ public void testAddSystemTopic() {
         assertThat(TopicValidator.getSystemTopicSet()).contains(topic);
     }
 
+    @Test
+    public void testValidateSystemTopic() {
+        RemotingCommand response = RemotingCommand.createResponseCommand(-1, "");
+
+        String topic = TopicValidator.RMQ_SYS_SCHEDULE_TOPIC;
+        Boolean res = TopicValidator.validateSystemTopic(topic, response);
+        assertThat(res).isFalse();
+        assertThat(response.getCode()).isEqualTo(ResponseCode.SYSTEM_ERROR);
+        assertThat(response.getRemark()).isEqualTo("The topic[" + topic + "] is conflict with system topic.");
+
+        topic = "test_topic";
+        res = TopicValidator.validateSystemTopic(topic, response);
+        assertThat(res).isTrue();
+    }
+
+    @Test
+    public void testValidateBlacklistTopic() {
+        RemotingCommand response = RemotingCommand.createResponseCommand(-1, "");
+
+        String topic = TopicValidator.RMQ_SYS_SCHEDULE_TOPIC;
+        Boolean res = TopicValidator.validateBlacklistTopic(topic, response);
```

Comment:
it still needs to polish when you change your blacklist method in validator class.